core/services/vrf/vrf_simulate_blockchain_test.go

```diff
@@ -116,3 +119,91 @@ func TestIntegration_RandomnessRequest(t *testing.T) {
 	})
 	require.NoError(t, err, "problem verifying solidity proof")
 }
+
+// TestIntegration_SharedProvingKey tests the scenario where multiple nodes share
+// a single proving key
+func TestIntegration_SharedProvingKey(t *testing.T) {
+	config, _, cfgCleanup := cltest.BootstrapThrowawayORM(t, "vrf_shared_proving_key_integration_test", true, true)
```

Comment:
Every db name is prepended with "chainlink_test" so including "_integration_test" here is probably superfluous.